examples/.experimental/next-app-dir/src/app/layout.tsx

```diff
@@ -20,8 +22,19 @@ export default function RootLayout({
           style={{
```

Comment:
might have to add tailwind here soon cause these big style blocks are annoying 😓

Reply:
do it if you wna!